main.go

```diff
@@ -100,6 +101,8 @@ func init() {
 	flag.BoolVar(&config.S7, "s7", false, "Send some Siemens S7 data")
 	flag.BoolVar(&config.NoSNI, "no-sni", false, "Do not send domain name in TLS handshake regardless of whether known")
 
+	flag.StringVar(&clientHelloFileName, "raw-client-hello", "", "Provide a raw ClientHello to be sent; only the SNI will be rewritten (implies --tls)")
```

Comment:
I don't think this actually implies `--tls` as written, which is fine. I'd just remove the text here.